src/dmd/json.d

```diff
@@ -826,8 +836,61 @@ public:
     private void generateCompilerInfo()
     {
         objectStart();
-        property("binary", global.params.argv0);
-        property("version", global._version);
+        propertyOrNull("vendor", global.compiler.vendor);
+        propertyOrNull("version", global._version);
+        property("__VERSION__", global.versionNumber());
+        propertyOrNull("interface", determineCompilerInterface());
+        property("bitWidth", size_t.sizeof * 8);
```

Comment:
Not sure if `bitWidth` is a good name.

Reply:
yeah, any other ideas?  It's the same number that's printed in the --version output.

Reply:
`size_tSizeof`

Reply:
ok, `size_t` seems reasonable.  With `size_t` I also changed it to the "byte-width" instead of "bit-width".   Let me know if someone prefers it to be the bit-width instead.